ledger/participant-integration-api/src/main/scala/platform/indexer/JdbcIndexer.scala

```diff
@@ -187,15 +187,15 @@ object JdbcIndexer {
           .map(
             info =>
               Map(
-                "updateSubmitter" -> info.submitter,
+                "updateSubmitter" -> info.actAs.mkString("[", ", ", "]"),
```

Comment:
Perhaps we can introduce a helper function for this, so implementations don't diverge in the future?